test/test_torch.py

```diff
@@ -7313,6 +7313,10 @@ def test_roll(self):
             self.assertEqual(expected, rolled,
                              "non contiguous tensor rolled to {} instead of {} ".format(rolled, expected))
 
+            # test roll with no dimension specified
+            expected = numbers.roll(1, 0).view(2, 4)
```

Comment:
this test looks good -- can you also add an empty test as you did before?

Reply:
added